### PR TITLE
Debug info + atomic ints

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: pkg/sfu/downtrack.go [hunk 1]
```diff
@@ -213,7 +223,7 @@ func (d *DownTrack) SwitchSpatialLayer(targetLayer int32, setAsMax bool) error {
 	}
 
 	// already set
-	if d.CurrentSpatialLayer() == targetLayer {
+	if d.currentSpatialLayer.get() == targetLayer {
```
Comment (davidzhao) [inline pkg/sfu/downtrack.go:226]: nit: seems more readable with the function wrapper vs .get

File: pkg/sfu/receiver.go [hunk 2]
```diff
@@ -454,10 +455,12 @@ func (w *WebRTCReceiver) writeRTP(layer int32, dt *DownTrack, pkt *buffer.ExtPac
 				dt.SwitchSpatialLayerDone(targetLayer)
 				currentLayer = targetLayer
 			} else {
+				dt.lastPli.set(time.Now().UnixNano())
```
Comment (davidzhao) [inline pkg/sfu/receiver.go:458]: why is this needed? wouldn't `SendRTCP` set this?

**Review by davidzhao — COMMENTED** [review]
lg! left a couple of Qs

File: pkg/sfu/receiver.go (hunk 2 above)
Comment (frostbyte73) [reply pkg/sfu/receiver.go:458]: SendRTCP sets the receiver's `lastPli`, we also track it per down track

Comment (davidzhao) [reply pkg/sfu/receiver.go:458]: ah I missed it's on dt. sg
